src/products/components/ProductMediaNavigation/ProductMediaNavigation.tsx

```diff
@@ -62,25 +67,34 @@ const ProductImageNavigation: React.FC<ProductImageNavigationProps> = props => {
     <Card className={classes.card}>
       <CardTitle
         title={intl.formatMessage({
-          defaultMessage: "All Photos",
+          defaultMessage: "All Media",
           description: "section header"
         })}
       />
       <CardContent>
-        {images === undefined ? (
+        {media === undefined ? (
           <Skeleton />
         ) : (
           <div className={classes.root}>
-            {images.map(image => (
+            {media.map(mediaObj => (
               <div
                 className={classNames({
                   [classes.imageContainer]: true,
-                  [classes.highlightedImageContainer]: image.id === highlighted
+                  [classes.highlightedImageContainer]:
+                    mediaObj.id === highlighted
                 })}
-                onClick={onRowClick(image.id)}
-                key={image.id}
+                onClick={onRowClick(mediaObj.id)}
+                key={mediaObj.id}
               >
-                <img className={classes.image} src={image.url} />
+                <img
+                  className={classes.image}
+                  src={
+                    mediaObj.type && mediaObj.type !== ProductMediaType.IMAGE
```

Comment:
Could be shortened to mediaObj?.type !== ProductMediaType.IMAGE ?
Also, why use the negative condition for this instead of reversing it?
```suggestion
                    mediaObj.type && mediaObj.type === ProductMediaType.IMAGE
```
Also since this is used multiple times in multiple places it seems like it deserves a helper function in utils somewhere